src/main/webapp/javascript/imageDisplayTools.js

```diff
@@ -139,9 +139,9 @@ maLib.maJsonToFigureElemCaption = function(maJson, intoElem, caption, maCaptionF
   if (!wildbook.user.isAnonymous()) {
 
   	fig.append('<figcaption itemprop="caption description">'+caption+maCaptionFunction(maJson)+'</figcaption>');
-    // fig = updateWithAnnotationDisambiguator(fig);
   }
   intoElem.append(fig);
+  intoElem = updateWithAnnotationDisambiguator(intoElem, maJson.id);
```

Comment:
not sure you need to _assign_ `intoElem` to the return of this function, as it is being passed in and modified within it.  it makes me a little nervous; but i suppose if it is working that is fine!  :)